docs/source/connecting/connecting.md

```diff
@@ -34,6 +34,52 @@ async fn main() -> Result<(), Box<dyn Error>> {
 After successfully connecting to some specified node the driver will fetch topology information about
 other nodes in this cluster and connect to them as well.
 
+Scylla Serverless is an elastic and dynamic deployment model. When creating a `Session` you need to
+specify the secure connection bundle as follows:
+
+```rust
```

Comment:
The CI fails because it compiles and runs examples from the documentation, and this particular example tries to use a yaml configuration that doesn't exist to actually connect to Scylla Cloud.

AFAIK we don't simulate Scylla Cloud in CI, so until we do we should use the `no_run` [attribute](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#attributes):

    ```rust,no_run
    // Contents of the example go here
    ```

Reply:
> As @cvybhu suggested, the example is wrapped under a `check_only_compiles` function which ensures the example is compiled but not run.

Ok, such a solution makes sense as well.